transcom/mymove
# [ MB-16172 ] Payment service item review should show same information as SIT Dashboard

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16172)

## Summary

This PR changes the calculations in `payment_request/payment_request_shipments_sit_balance.go` to utilize the functions in `shipment_sit_status.go` that calculate  `totalSITDaysAuthorized`, `totalSITDaysRemaining`, and `totalSITEndDate`. 

Here is a link to how to replicate the issue: https://dp3.atlassian.net/wiki/spaces/~62dff422bc2c449f3d9403fd/pages/2213412919/MB-16172+notes#How-to-replicate-the-issue

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Take an approved move and create a SIT service item. The bug in staging is seen on a Destination SIT service item, but origin should cause the same bug. I used move code `SITEXT`. It already has a SIT extension requested and origin SIT service items, so for this one you would want to create Destination SIT service items.
2. Go in as a TOO and approve those service items. 
3. Using Postman or in the prime simulator, create multiple payment requests for the DDASIT service item. Make sure you have non overlapping dates. Here is what that looks like in Postman:
![image](https://github.com/transcom/mymove/assets/110134470/58aed9f0-2f1a-41f8-a41e-c6122127ff66)
4. Log in as the TOO and view the SIT Dashboard (on the Move task order tab). This is what you will use for comparison in the next step. Note the total days remaining and the sit authorized end date: 
![image](https://github.com/transcom/mymove/assets/110134470/7d712515-3f81-4b18-8241-8530ad650621)
5. Log in as a TIO and go to the move in question. There should be multiple review service items buttons. 
![image](https://github.com/transcom/mymove/assets/110134470/46baf08a-6422-41de-8786-e84998cf2615)
6. For each review button you click, you will view a different payment request. The Authorized remaining days and the end date should now match the information from the SIT Dashboard. 
![image](https://github.com/transcom/mymove/assets/110134470/85911ace-2cc3-4672-89d1-18f2ea3c7fc2)



### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
